tests/pandas_postprocessing_tests.py

```diff
@@ -508,3 +509,119 @@ def test_contribution(self):
         self.assertListEqual(df.columns.tolist(), ["a", "b"])
         self.assertListEqual(series_to_list(column_df["a"]), [0.25, 0.75])
         self.assertListEqual(series_to_list(column_df["b"]), [0.1, 0.9])
+
+    def test_prophet_incorrect_values(self):
```

Comment:
Breaking this up into a test for each incorrect value type may make debugging test failures easier down the road.

Reply:
Good idea, will break this up.